projects/plugins/beta/src/class-hooks.php

```diff
@@ -595,7 +609,11 @@ public static function custom_error_handler( $errno, $errstr, $errfile, $errline
 			}
 		}
 
-		// Returning false makes the error go through the standard error handler as well.
-		return false;
+		if ( self::$prev_error_handler ) {
+			return call_user_func( self::$prev_error_handler, $errno, $errstr, $errfile, $errline );
+		} else {
```

Comment:
Nit: I guess we could drop the else in this case and just have `return false;`. Just a personal preference in the spirit of keeping things simple, it's up to you ;)

Reply:
Doesn't matter to me. But looks like the auto-merge already auto-merged.